sdk/attestation/attestation/samples-dev/attestEnclaves.ts

```diff
@@ -0,0 +1,368 @@
+    console.log("Relying party validates: ");
+    console.log(" Enclave Hash: ", attestResponse.body.mrEnclave);
+    console.log(" Enclave Signer: ", attestResponse.body.mrSigner);
+    console.log(" Enclave security version: ", attestResponse.body.svn);
+    console.log(" Enclave product ID:", attestResponse.body.productId);
+    if (attestResponse.body.enclaveHeldData) {
+      console.log(
+        " Runtime Data received by the service: ",
+        bytesToString(attestResponse.body.enclaveHeldData)
+      );
+    }
+  }
+}
+
+async function attestOpenEnclaveWithRuntimeJson() {
+  writeBanner("Attest Open Enclave With Runtime JSON.");
+  const endpoint = process.env.ATTESTATION_AAD_URL;
+  if (!endpoint) {
+    throw new Error("ATTESTATION_AAD_URL must be provided.");
+  }
+
+  console.log("Attest an OpenEnclave enclave from: ", endpoint);
+  const client = new AttestationClient(new DefaultAzureCredential(), endpoint);
+
+  const openEnclaveReport = decodeString(_openEnclaveReport);
+
+  // The data contained in the _runtimeData encoded blob is a JSON Web Key, so we
+  // can interpret it as JSON data.
+  const runtimeJson = decodeString(_runtimeData);
+
+  // Attest an SGX enclave with Runtime Json. Runtime Json can be a public key
+  // generated from within the enclave. The attestation service will determine
+  // that the SGX Quote (evidence) contains the hash of the Runtime Data and if
+  // it does, it will emit a token for the quote.
+  {
+    const attestResponse = await client.attestOpenEnclave(openEnclaveReport, {
+      runTimeJson: runtimeJson
+    });
+
+    console.log("Received attestation token: ", attestResponse.token.serialize());
+
+    // The attestation token above can be sent to a relying party who will validate
+    // the contents of the token. They will then look at the runtimeClaims
+    // and treat those runtimeClaims as a JSON object. In this case, the JSON object
+    // contains a RFC 7517 JSON Web Key which can be used to encrypt data to be
+    // transmitted to the enclave.
+    console.log("Relying party validates: ");
+    console.log(" Enclave Hash: ", attestResponse.body.mrEnclave);
+    console.log(" Enclave Signer: ", attestResponse.body.mrSigner);
+    console.log(" Enclave security version: ", attestResponse.body.svn);
+    console.log(" Enclave product ID:", attestResponse.body.productId);
+    if (attestResponse.body.runTimeClaims) {
+      const jwk = attestResponse.body.runTimeClaims as {
+        jwk: { kty: string; use: string; crv: string; x: string; y: string };
+      };
+
+      console.log("  Received JWK is: ", jwk.jwk.kty);
+      console.log("  Received JWK uses the following curve ", jwk.jwk.crv);
+    }
+  }
+}
+
+async function attestOpenEnclaveWithExperimentalPolicy() {
+  writeBanner("Attest Open Enclave with Experimental Policy.");
+  const endpoint = process.env.ATTESTATION_AAD_URL;
+  if (!endpoint) {
+    throw new Error("ATTESTATION_AAD_URL must be provided.");
+  }
```

Comment:
NIT: you're loading `ATTESTATION_AAD_URL` multiple times, can we load it once in the main function and pass it as param instead?

Reply:
I thought of that, but my intention for samples is that each sample is as standalone as I can make it. That means I try to avoid state which is external to the sample methods (except for helper functions in the utils directory).